plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoMetadata.java

```diff
@@ -178,6 +217,15 @@ private List<SchemaTableName> listTables(ConnectorSession session, SchemaTablePr
     @Override
     public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTableHandle tableHandle, ColumnHandle columnHandle)
     {
+        MongoTableHandle table = (MongoTableHandle) tableHandle;
```

Comment:
Since we capture `FILTER` to be applied on a collection - can we reuse the same column metadata of that of a collection ?